src/core/Track.cpp

```diff
@@ -317,6 +319,12 @@ TrackContentObjectView::~TrackContentObjectView()
  */
 void TrackContentObjectView::update()
 {
+	if( !m_cursorSetYet )
```

Comment:
I only set this once to avoid setting the cursor all the time. I hope this is enough, but the mouse hotspot should actually never change during runtime, right?

Reply:
It can be set when a cursor is constructed, isn't that enough?

Reply:
Nevermind, I missed your comment in the issue post.

Reply:
What do you mean? The cursor has already been constructed in `TrackContentObjectView`'s CTOR, and at that time, the hotspot is not yet read from the CSS.